config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
```

Comment:
This was updated by the make bundle target as we use different ns for upstream and downstream.